vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Handling Usage Files
+description: Guidance for handling usage files.
+ms.date: 05/11/2023
+---
+
+# Handling Usage Files
+
+## Overview
+Usage files play an important role in providing instructions, examples, or guidelines for utilizing packages within our repository. To ensure consistency and clarity, we have established the following guidelines for handling usage files:
+
+**Using `configure_file()` Function**
```

Comment:
These should probably be H3s, not bold.
```suggestion
### Using `configure_file()` Function
```

However I also don't think this needs a section banner.